nav2_regulated_pure_pursuit_controller/src/parameter_handler.cpp

```diff
@@ -106,9 +119,16 @@ ParameterHandler::ParameterHandler(
     rclcpp::ParameterValue(true));
   declare_parameter_if_not_declared(
       node, plugin_name_ + ".stateful", rclcpp::ParameterValue(true));
+  declare_parameter_if_not_declared(
+      node, plugin_name_ + ".use_dynamic_window", rclcpp::ParameterValue(true));
+  declare_parameter_if_not_declared(
+      node, plugin_name_ + ".velocity_feedback", rclcpp::ParameterValue(std::string("OPEN_LOOP")));
```

Comment:
Why is this adding a new feature? Please remove and open another PR to add.

Reply:
It is much easier for maintainers to review 2x decoupled smaller PRs than 1 larger PR